src/msg/xio/XioConnection.cc

```diff
@@ -477,7 +477,7 @@ void XioConnection::msg_release_fail(struct xio_msg *msg, int code)
     " (" << xio_strerror(code) << ")" << dendl;
 } /* msg_release_fail */
 
-int XioConnection::flush_input_queue(uint32_t flags) {
```

Comment:
ok--this is a comment on the whole commit--why is this a misleading name?
my naming had "input" queue for queuing before any attempt to call xio_send_msg, and "out" queue for msgs we need to retry on xio_send_msg (iirc)